pkg/apis/apps/v1alpha1/apimanager_types.go

```diff
@@ -172,13 +172,20 @@ type BackendSpec struct {
 	// +optional
 	RedisImage *string `json:"redisImage,omitempty"`
 	// +optional
+	RedisPersistentVolumeClaimSpec *BackendRedisPersistentVolumeClaimSpec `json:"redisPersistentVolumeClaim,omitempty"`
```

Comment:
what about `json:"persistentVolumeClaim,omitempty"` ??

Reply:
I don't want to give it a "generic" name being in a broad section. If it were to be inside a "redis" section I'd do it but I didn't want to being there. It's the same case as with 'redisImage'. I didn't choose 'image' for 'redisImage' as it was in a shared broad section